event-based-simulators/main/scripts/README.md

```diff
@@ -0,0 +1,32 @@
+# Generic scripts information
+This scripts folder contains:
+- The **ExportOrImportProfileData.py** which is used to export measurements or alarms data from devices
+- The **Environment.py** which is used to setup environment parameters
```

Comment:
- since it is a config file and not a python file, the extensions should be .conf or .properties

Reply:
.py format is the easiest solution since it can be input directly into another .py script. Also if this script is used in the same Docker environment as the oee-simulators, it needs to adapt the environment setup of the oee-simulators. Therefore, before we can decide how to use this script (by users, by another scripts with command line or by oee-simulators in Docker container), I will keep this way for testing setup